tensorflow-framework/src/main/java/org/tensorflow/framework/regularizers/L1L2.java

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=======================================================================*/
+package org.tensorflow.framework.regularizers;
+
+import org.tensorflow.Operand;
+import org.tensorflow.framework.losses.impl.LossesHelper;
+import org.tensorflow.op.Ops;
+import org.tensorflow.types.family.TNumber;
+
+/**
+ * A regularizer that applies both L1 and L2 regularization penalties.
+ *
+ * <p>The L1 regularization penalty is computed as:
+ *
+ * <pre>loss = l1 * reduceSum(abs(x))</pre>
+ *
+ * <p>The L2 regularization penalty is computed as
+ *
+ * <pre>loss = l2 * reduceSum(square(x))</pre>
+ *
+ * <p>The difference between this class and the {@link L1_L2} is use of the default regularization
+ * penalty {@link #DEFAULT_REGULARIZATION_PENALTY}, whereas {@link L1L2} defaults to 0.
+ *
+ * @param <R> the data type for the weights
+ */
+public class L1L2<R extends TNumber> extends Regularizer<R> {
+
+  private final Float l1;
```

Comment:
Basically, `L1L2`, besides standing on its own, is the super class for `L1_L2`, `L1`, and `L2`.  We still need the Float for the constructor because null is a valid value for either `l1`, or `l2`, depending in the subclass.
However, I will change the internal representation to float.

Reply:
On second though, maybe we don't need the `Float`. Just change the ctors for `L1` and `L2` to pass `0` for the unused calculation.

Reply:
👍 done